tests/perf_unicorn.py

```diff
@@ -33,6 +36,9 @@ def perf_unicorn_1():
     pg_unicorn.run()
     elapsed = time.time() - start
 
+    if len(pg_unicorn.errored) > 0:
+        pg_unicorn.errored[0].debug()
+
```

Comment:
Can you remove this and the above check? We'd like our tests to not hang on error :)